packages/blade/src/components/SkipNav/__tests__/SkipNav.web.test.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import userEvents from '@testing-library/user-event';
+import renderWithTheme from '../../../_helpers/testing/renderWithTheme.web';
+import { SkipNavContent, SkipNavLink } from '../SkipNav.web';
+
+describe('<SkipNav />', () => {
+  it('<SkipNavLink /> should render correctly', () => {
+    const { getByRole } = renderWithTheme(<SkipNavLink />);
+
+    expect(getByRole('link')).toHaveTextContent('Skip to content');
+  });
+
+  it('<SkipNavLink /> should render correctly with children', () => {
+    const { getByRole } = renderWithTheme(<SkipNavLink children="Skip" />);
+
+    expect(getByRole('link')).toHaveTextContent('Skip');
+  });
+
+  it('<SkipNavContent /> should render correctly', () => {
+    const { getByTestId } = renderWithTheme(<SkipNavContent />);
+
+    expect(getByTestId('skipnav-content')).toHaveAttribute('tabindex', '-1');
+  });
+
+  it('should skip to main navigation', async () => {
+    jest.useRealTimers();
+    const user = userEvents.setup();
+    const { getByRole, getByTestId } = renderWithTheme(
+      <div>
+        <nav>
+          <SkipNavLink id="custom-id" />
+          <button>one</button>
+          <button>two</button>
+        </nav>
+        <main>
+          <SkipNavContent id="custom-id" />
+          <button>main content</button>
+        </main>
+      </div>,
+    );
+
+    // TODO: move to ariakit-utils after this is fixed:
+    // https://github.com/ariakit/ariakit/issues/1483
+    await user.tab();
+    expect(getByRole('link')).toHaveFocus();
```

Comment:
let's add an assertion for the text content too? because we shall also be sure right that we have focus on the correct link

Reply:
I'm already doing text content assertion in above two test cases. Don't think it's necessary again. 
We are sure the focus is on the correct link, since we only have one link in the example JSX inside the renderWithTheme

Reply:
I see, but then shouldn't we have a test to add maybe 2 SkipNav components like you have in the storybook? that will give us the actual confidence IMO

Reply:
Oh yeah that makes sense. I'll add a test for it.